tst/EnergyPlus/unit/Furnaces.unit.cc

```diff
@@ -87,6 +87,7 @@ namespace EnergyPlus {
 TEST_F(EnergyPlusFixture, SetVSHPAirFlowTest_VSFurnaceFlowTest)
 {
 
+    state->init_state(*state);
```

Comment:
Again a little curious/suspicious why just a few specific tests need this added.  I expected either it be added to the fixture base class and no tests would need it, or every test would need it.